internal/controllers/symlink-resolver/resolver.go

```diff
@@ -0,0 +1,43 @@
+package symlinkResolver
```

Comment:
I suggest that all of the imports also use this case instead of _ . Waiting for input from @suleymanakbas91